include/config/windowswitcher-field.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef __LABWC_WINDOWSWITCHER_FIELD_H
+#define __LABWC_WINDOWSWITCHER_FIELD_H
+
+struct windowswitcher_field {
+	char *content;
+	int width;
+	struct wl_list link; /* struct rcxml.windowswitcher.fields */
+};
```

Comment:
I am not sure if we need the new header file here, we could also define this in the generic `rcxml.h` header.

Reply:
Have removed new header file and moved struct to `rcxml.h`